templates/serviceaccount.yaml

```diff
@@ -5,14 +5,18 @@
 apiVersion: 'v1'
 kind: 'ServiceAccount'
 metadata:
-  name: {{ include "media-servarr-base.serviceAccount.name" . }}
+  name: {{ include "media-servarr-base.serviceAccountName" . }}
```

Comment:
In the basic [values.yaml](https://github.com/drinkataco/media-servarr/blob/main/values.yaml#L121) this is set as serviceAccount.name, not serviceAccountName

Reply:
And the helper is also the one used within the deployment.yaml and the names between service account and deployment.yaml shouldn't be different.

Reply:
Oops, my bad. I misread the change!